dev/tests/integration/testsuite/Magento/Framework/Interception/PluginListGeneratorTest.php

```diff
@@ -94,7 +94,6 @@ public function testPluginListConfigGeneration()
             1 => [
                 0 => 'genericHeaderPlugin',
                 1 => 'asyncCssLoad',
```

Comment:
@ihor-sviziev seem after this change plugin name asyncCssLoad already changed to intercept class 
Magento\Framework\View\Result\Layout instead previously Magento\Framework\App\Response\Http. So we should remove item asyncCssLoad. Is that correct ?